### PR TITLE
Add missing ActivityFields and override getFieldsEnum in Activity.

### PR DESCRIPTION
**Pull request by xplinscott**

### CODE REVIEW

File: src/FacebookAds/Object/Fields/ActivityFields.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright 2015 Facebook, Inc.
```
Comment (pruno) [inline src/FacebookAds/Object/Fields/ActivityFields.php]: Year should be 2014

File: src/FacebookAds/Object/Fields/ActivityFields.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    const ACTOR_ID = 'actor_id';
+    const ACTOR_NAME = 'actor_name';
+    const EXTRA_DATA = 'extra_data';
+    const EVENT_TIME = 'event_time';
+    const EVENT_TYPE = 'event_type';
+    const OBJECT_ID = 'object_id';
+    const OBJECT_NAME = 'object_name';
+}
+
```
Comment (pruno) [inline src/FacebookAds/Object/Fields/ActivityFields.php]: Can you pls remove this trailing empty line?

**Comment by pruno** [comment]
Thanks @xplinscott, this looks great, can you please address the 2 nit inline comments?

**Comment by xplinscott** [comment]
Hi @pruno, the commit has been amended with your requested changes. Thanks!

**Comment by pruno** [comment]
Thanks!

**Pull request merged**